### PR TITLE
Fix #297: Remove project powerauth-restful-server-spring

### LINKED ISSUE #297: Remove project powerauth-restful-server-spring
Let's remove the `powerauth-restful-server-spring` project, it confuses customers because of its similarity to the `enrollment-server` component. We should move the controllers and providers which are used for testing to a Wultra fork of enrollment server.

### PR DESCRIPTION
**Pull request by romanstrobl**
This pull request removes `powerauth-restful-server-spring` and updates the documentation so that `enrollment-server` is deployed instead. We will use a fork of Enrollment server for testing purposes, I will move the mock controllers and providers into a new project based on Enrollment server.

### CODE REVIEW

File: docs/Deploying-PowerAuth-Standard-RESTful-API.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+## Downloading Enrollment Server
 
-You can download the latest `powerauth-restful-server.war` at the releases page:
+You can download the latest `enrollment-server.war` at the releases page:
 
-- https://github.com/wultra/powerauth-restful-integration/releases
+- https://github.com/wultra/enrollment-server/releases
 
-## Configuring PowerAuth Standard RESTful API
+_Note: The enrollment server component will need to be customized in case you need to customize the activation process. The customization is described in the tutorial [Implementing the Server-Side for Authentication in Mobile Banking Apps (SCA)](https://developers.wultra.com/products/mobile-token/2021-05/tutorials/Authentication-in-Mobile-Apps/Server-Side-Tutorial#deploying-the-enrollment-server)._
```
Comment (petrdvorak) [inline docs/Deploying-PowerAuth-Standard-RESTful-API.md:15]: Use the DocuCheck metadata here:

```xml
<-- begin box warning -->
<-- end -->
```

Let's edit this across this documention.

File: docs/Deploying-PowerAuth-Standard-RESTful-API.md [hunk 2]
```diff
@@ -1,20 +1,22 @@
-# Deploying PowerAuth Standard RESTful API Bootstrap
+# Deploying PowerAuth RESTful API
```
Comment (petrdvorak) [inline docs/Deploying-PowerAuth-Standard-RESTful-API.md:1]: Can we move this to the "Enrollment Server" documentation completely / merge it with the Docs there?

**Review by petrdvorak — APPROVED** [review]
Looks OK, I would just prefer to see the documentation related to Enrollment Server at the Enrollment Server repository...

File: docs/Deploying-PowerAuth-Standard-RESTful-API.md (hunk 1 above)
Comment (romanstrobl) [reply docs/Deploying-PowerAuth-Standard-RESTful-API.md:15]: Ok, I will add it.

File: docs/Deploying-PowerAuth-Standard-RESTful-API.md (hunk 2 above)
Comment (romanstrobl) [reply docs/Deploying-PowerAuth-Standard-RESTful-API.md:1]: Yes, I will move the documentation.

**Pull request merged**